index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <title>Portofolio</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navigation">
+        <h1 id="wlcm">Welcome</h1>
+        <ul class="nav-list">
+          <li>
+            <a href="#">Portofolio</a>
+          </li>
+          <li>
+            <a href="#">About</a>
+          </li>
+          <li>
+            <a href="#">Contact</a>
+          </li>
+        </ul>
+        <i class="far fa-bars"></i>
+      </nav>
+    </header>
+    <section id="welcome-section">
+      <div class="center">
+        <h1 class="name">Hey there. I'm Zachee</h1>
+        <h2 class="title">I am a software developer</h2>
+        <p class="description">
+          I can help you build a product, feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestate to Contact me.
+        </p>
+        <div class="icons">
+          <i class="fab fa-github-square"></i>
+          <i class="fab fa-linkedin-in"></i>
+          <i class="far fa-hand-peace"></i>
+          <i class="fab fa-twitter"></i>
+          <i class="fab fa-medium-m"></i>
+        </div>
```

Comment:
The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag).